src/remote-config.ts

```diff
@@ -54,6 +55,7 @@ export const defaultRemoteConfig: RemoteConfig = {
   preassigned_fare_products: JSON.stringify(defaultPreassignedFareProducts),
   tariff_zones: JSON.stringify(defaultTariffZones),
   user_profiles: JSON.stringify(defaultUserProfiles),
+  customer_service_url: 'https://www.atb.no/kundeservice/',
```

Comment:
Har hatt en diskusjon med kundeservice her, og de mener at siden kunden har tatt et konkret valg om å kontake kundeservice, burde URL-en være https://www.atb.no/kontakt/

```suggestion
  customer_service_url: 'https://www.atb.no/kontakt/',
```